diagrams/theme.json

```diff
@@ -13,13 +13,44 @@
       "shape": "Person"
     },
     {
-      "tag": "Highlighted",
+      "tag": "FocusedSystem",
```

Comment:
Given the active containers and components are a different shade of blue. Using "Highlighted" was too generic here.

Reply:
All the other tags are phrases so this should be `Focused System`.

Also, the change from `Highlighted` to `Focused System` seems to imply that this tag should only be added to `softwareSystem`s. Is that the intention?

Reply:
Yes the intent is it's just for the software System.

Most of them are not. The two other that are is only because I copied that from the default theme, which I assumed was a special keyword.

Reply:
Chatted.

Structurizr uses tags as labels in the legend. So tags should be more human readable than machine readable.